circuit_knitting/cutting/cutting_experiments.py

```diff
@@ -273,19 +305,29 @@ def _append_measurement_circuit(
                 f"qubit_locations has {len(qubit_locations)} element(s) but the "
                 f"observable(s) have {cog.general_observable.num_qubits} qubit(s)."
             )
+
+    # Find observable_measurements register
+    for reg in qc.cregs:
+        if reg.name == "observable_measurements":
+            obs_creg = reg
+            break
+    else:
```

Comment:
Neat code here where the else sits outside the looped if ... I'll have to glance at how this works :)